x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/tables/meta_engines_table_expanded_row.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiBasicTable, EuiHealth, EuiTextAlign, EuiTitle } from '@elastic/eui';
+
+import { EngineDetails } from '../../../engine/types';
+import { SOURCE_ENGINES_TITLE } from '../../constants';
+
+import {
+  BLANK_COLUMN,
+  CREATED_AT_COLUMN,
+  DOCUMENT_COUNT_COLUMN,
+  FIELD_COUNT_COLUMN,
+  NAME_COLUMN,
+} from './shared_columns';
+
+import './meta_engines_table_expanded_row.scss';
+
+interface MetaEnginesTableExpandedRowProps {
+  sourceEngines: EngineDetails[];
+  conflictingEngines: Set<string>;
+}
+
+export const MetaEnginesTableExpandedRow: React.FC<MetaEnginesTableExpandedRowProps> = ({
+  sourceEngines,
+  conflictingEngines,
+}) => (
+  <div className="meta-engines__source-engines-table">
+    <EuiTextAlign textAlign="center">
+      <EuiTitle size="xs">
+        <h3>{SOURCE_ENGINES_TITLE}</h3>
+      </EuiTitle>
+    </EuiTextAlign>
```

Comment:
optionally, you could also use EUI's CSS utility classes for text alignment if you don't want an extra wrapper

```suggestion
    <EuiTitle size="xs" className="eui-textCenter">
      <h3>{SOURCE_ENGINES_TITLE}</h3>
    </EuiTitle>
```